readme.md

```diff
@@ -120,26 +149,48 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:**
 
+Model.findById()
+
+https://mongoosejs.com/docs/api/model.html#model_Model.findById
+
 ---
 
 - [ ] Update the API so that you are able to update a document. Use the [Model.updateOne()](https://mongoosejs.com/docs/api.html#model_Model.updateOne) method.
 
 * **Question:** What are the arguments for `Model.updateOne()`?
 
 * **Your Answer:**
+filter - updates the first document that matches this.
+
+doc
+
+options (optional)
+
+callback - a function.  (error, writeOpResult)
+
+
 
 * **Question:** The response you receive is _not_ the document you updated. What information is being represented here? Try replacing `Model.updateOne()` with the [Model.findOneAndUpdate()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndUpdate) and see the difference in the result.
 
 * **Your Answer:**
 
+(I coded both calls.)
+
+updateOne returns the _old document_. The other method returns the new document.
```

Comment:
This is not quite true. `.updateOne()` will return meta information about the update, while `.findOneAndUpdate()` will return the old document.